cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -441,13 +449,23 @@ mkConsensusTracers
   -> ForgingStats
   -> IO (Consensus.Tracers' peer localPeer blk (Tracer IO))
 mkConsensusTracers trSel verb tr nodeKern fStats = do
+  let trmet = appendName "metrics" tr
+
   blockForgeOutcomeExtractor <- mkOutcomeExtractor
   elidedFetchDecision <- newstate  -- for eliding messages in FetchDecision tr
   forgeTracers <- mkForgeTracers
+  meta <- mkLOMeta Critical Public
+
+  headerMessageCount <- STM.newTVarIO 0
```

Comment:
@deepfire I'm looking for a way to avoid having to create a TVar here and let the monitoring framework handle the counting.

Reply:
@newhoggy, question is -- we have multiple peers we're syncing from, so a thread-local counter wouldn't make a lot of sense, which means you'd need shared state anyway.

Or do I misunderstand your intent here?

Reply:
That makes sense.  I'm happy to keep the shared state.